packages/data-stores/src/reader/queries/use-subscription-manager-site-subscriptions-query.ts

```diff
@@ -0,0 +1,71 @@
+import { useQuery } from 'react-query';
+import { callApi } from '../helpers';
+import { useIsLoggedIn, useIsQueryEnabled } from '../hooks';
+import type { SiteSubscription } from '../types';
+
+type SubscriptionManagerSiteSubscriptions = {
+	subscriptions: SiteSubscription[];
+	page: number;
+	total_subscriptions: number;
+};
+
+type SubscriptionManagerSiteSubscriptionsQueryProps = {
+	filter?: ( item?: SiteSubscription ) => boolean;
+	sort?: ( a?: SiteSubscription, b?: SiteSubscription ) => number;
+	limit?: number;
+};
+
+const callFollowingEndPoint = async (
+	page: number,
+	limit: number,
+	isLoggedIn: boolean
+): Promise< SiteSubscription[] > => {
+	const data = [];
+
+	const incoming = await callApi< SubscriptionManagerSiteSubscriptions >( {
+		path: `/read/following/mine?limit=${ limit }&page=${ page }`,
```

Comment:
While testing the query limit, I found one more thing. When looking at the API (2fb19-pb), it is expecting the `number` parameter instead of the `limit`. If we use `limit` only, the pagination won't work as expected. 🙂 

If the `number` parameter isn't provided, `$subs_query['limit']` will be set to `self::PER_PAGE_MAX` and all the records will be fetched at once (with the limit of `100` of course).

Reply:
Wo, that's a huge oversight on my behalf. Let me fix it.